core/src/main/java/org/akaza/openclinica/service/crfdata/EnketoUrlService.java

```diff
@@ -248,9 +250,16 @@ public QueriesBean buildQueryElement(ItemData itemdata) {
             }
             query.setComment(dn.getDetailedNotes());
             query.setStatus(dn.getResolutionStatus().getName().toLowerCase());
-            query.setDate_time(dn.getDateCreated().toString());
+
```

Comment:
joda.time is in maintenance mode. Please use java.time
For more info look at this link:
http://blog.joda.org/2014/11/converting-from-joda-time-to-javatime.html